garage/sampler/__init__.py

```diff
@@ -1,8 +1,3 @@
-from garage.sampler.stateful_pool import SharedGlobal
 from garage.sampler.stateful_pool import singleton_pool
-from garage.sampler.parallel_sampler import truncate_paths  # noqa: I100
```

Comment:
Can you move truncate paths into a util in sampler? It hardly belongs inside parallel_sampler.

Reply:
done